modules/base/mw.cx.SiteMapper.js

```
@@ -159,7 +176,8 @@ mw.cx.SiteMapper = class {
 				.then( ( response ) => response.json() )
 				.then( ( response ) => ( {
 					targetLanguages: response.target,
-					sourceLanguages: response.source
+					sourceLanguages: ["en", "ar"]
```

ESLint

[error] A space is required after '['.

(array-bracket-spacing)